Makefile

```diff
@@ -1,14 +0,0 @@
-pack:
-	./build.sh
-	sort files > files.sorted
```

Comment:
What's the new recommended way to build the .xpi file to be shipped?

Reply:
never used the makefile, and jar is no longer allowed.

Zip with 7zip and rename from zip to xpi.